math/abella/Portfile

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           ocaml 1.1
+
+name                abella
+version             2.0.7
+revision            0
+categories          math ocaml
+maintainers         {@barracuda156 gmail.com:vital.had} {pmetzger @pmetzger} openmaintainer
+license             GPL-3
+description         Interactive theorem prover
+long_description    Abella is an interactive theorem prover based on lambda-tree syntax. \
+                    This means that Abella is well-suited for reasoning about the meta-theory \
+                    of programming languages and other logical systems which manipulate objects with binding.
+homepage            https://abella-prover.org
+master_sites        http://abella-prover.org/distributions
```

Comment:
any reason this isn't using https?

Reply:
@reneeotten I checked now, https works fine, changed to https.